### ISSUE TITLE
Align dashboard designs (with the website/signupflow/me app)

### ISSUE THREAD

**Issue by jamalv** [teamforus/general#215]
We have been working on harmonizing our designs. We are now starting to get a more consistent design-language between the website, me app and signupflows. With some (i think) minor tweaks (mainly background color and borders) we can already bring the dashboards much closer to the other designs.

## Before:
<img width="988" alt="Screenshot 2020-07-07 at 10 43 27" src="https://user-images.githubusercontent.com/30194799/86751087-b6775280-c03e-11ea-8cd8-2346371a9036.png">

## After:
![dev forus io_sponsor_](https://user-images.githubusercontent.com/30194799/82925313-d286cf00-9f7d-11ea-9bad-efa8af9a688d.png)

**Comment by jamalv** [comment]
![dev forus io_sponsor_](https://user-images.githubusercontent.com/30194799/82925313-d286cf00-9f7d-11ea-9bad-efa8af9a688d.png)

**Comment by GerbenBosschieter** [comment]
#161 
 This one is related

**Comment by jamalv** [comment]
@maxvisser how big is the scope for this? it seems like a minor issue, but now that our website, signupflows and the me app are more aligned in the white style, I think this would be a nice step in harmonising our products. 

Especcially if we start sharing style elements / images between the different products, it will help if the background colors are the same.

Example: ![image.png](https://images.zenhubusercontent.com/5bc6fc7b86f3a678d497cf05/efc7ed14-cc60-42dc-bdda-4c3120be502e)

Looks very different on a white background than on a light blue one :)

So, long story short, can this be planned?

**Comment by GerbenBosschieter** [comment]
@jamalv i agree. I would also like to see the white background on general webshop.🙉

**Comment by maxvisser** [comment]
> @maxvisser how big is the scope for this? it seems like a minor issue, but now that our website, signupflows and the me app are more aligned in the white style, I think this would be a nice step in harmonising our products.
> 
> Especcially if we start sharing style elements / images between the different products, it will help if the background colors are the same.
> 
> Example: ![image.png](https://camo.githubusercontent.com/628d9bcbe4e26f130daac031e8789fcb375a8712/68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3562633666633762383666336136373864343937636630352f65666337656431342d636336302d343264632d626464612d346333313230626535303265)
> 
> Looks very different on a white background than on a light blue one :)
> 
> So, long story short, can this be planned?

I think you mean a estimation of time? (which depends on the scope). If we want to align buttons, text fields, modals with the signupflow; each thing you add to this list adds upon how much time it takes. So maybe we can make a list on whats needed to be aligned.

**Comment by jamalv** [comment]
Ah yea, I understand. The scope I had in mind was purely the background color + all borders (remove dropshadows and make it the light grey line) - but I understand the scope can quickly grow with adding other UI elements as well..

for reference: https://www.figma.com/file/ArDg3BA8o4uANBLk3ZVzeM/Dashboards?node-id=65%3A138708

**Comment by jamalv** [comment]
of course, since this is mostly aesthetics and harmonisation, it will not be a "must-have" (we are not apple.. yet.. :P) - but making a start and improving it iteratively means we eventually increase re-usability of elements, which is an optimisation :)

**Comment by jamalv** [comment]
updated issue and title

**Comment by jamalv** [comment]
> @jamalv i agree. I would also like to see the white background on general webshop.🙉

Ah yes me too (https://webshop.staging.forus.io/), it would also be useful to align that more with the figma webshop library, as we often use those templates for designing webshops: https://www.figma.com/file/7lQX9ZRpJW3ZTCtEosAMzs/Webshop?node-id=156%3A0

**Comment by maxvisser** [comment]
I did some development during my free time on this issue. Its currently open as a pull-request. would be nice to update Figma aswell if there is time for this.

**Comment by maxvisser** [comment]
plan a white figma pizza party WHITE ALL THE THINGS

**Comment by jamalv** [comment]
closing this issue in favour of the pull request: https://github.com/teamforus/forus-frontend/pull/1482
